data/integral/trap.py

```diff
@@ -0,0 +1,13 @@
+import numpy
+
+def f(x):
+    return x ** 4 * np.log(x + np.sqrt(x ** 2 + 1))
+
+a = 0
+b = 5
+n = 100
+
+x = np.linspace(a,b,num=n)
+test_intgr = np.trapz(f(x), x=x)
+
+result = True if np.isclose(trap(f,n,a,b),test_intgr) else False
```

Comment:
This will always be True since np.close() always exist (gives an array of booleans). You need to use `np.close().all()` to return one boolean.

Reply:
np.isclose(1.22222,1.222225) returns a boolean, not a boolean array (despite the documentation).
It works.